x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -439,6 +446,7 @@ export const visualizationTypes: VisualizationType[] = [
     label: i18n.translate('xpack.lens.xyVisualization.barLabel', {
       defaultMessage: 'Bar',
```

Comment:
To be consistent with naming and match the designs.

```suggestion
      defaultMessage: 'Bar vertical',
```